en_US/advanced/relup.md

```diff
@@ -1,312 +1,173 @@
-i.e. upgrading from x.y.z to x.y.(z+N) is allowed, but upgrading from x.y to (x+N).(y+M) is not allowed.
-Please operate with caution.
+Currently EMQX for Windows and MacOS does not support the release hot upgrade feature.
 :::
 
 ::: warning
-Hot upgrade between EMQX (opensource) and EMQX Enterprise is not allowed, please operate with caution.
+EMQX only allows release hot upgrade between patch versions (the third digit of the version number). Please operate with caution.
+That is, upgrading from x.y.z to x.y.(z+N) is allowed, but upgrading from x.y to (x+N).(y+M) is not allowed.
+e.g. It is OK to upgrade to 4.4.16 from 4.4.1, 4.4.2, ..., but not from 4.3.1, 4.3.2, ...
 :::
 
-## Hot upgrade steps
-
-1. View the currently installed version list of EMQX.
-
-```bash
-
-$ emqx versions
-
-Installed versions:
-* 4.2.0	permanent
-```
-
-2. Download the software package to be upgraded from the EMQX website.
-
-Visit [EMQX](https://www.emqx.com/en/downloads?product=broker), Select the corresponding version and operating system type, and then select the **"zip"** package type.
-
-3. Find the installation directory of EMQX:
-
-```bash
-
-$ EMQX_ROOT_DIR=$(emqx root_dir)
-
-$ echo ${EMQX_ROOT_DIR}
-"/usr/lib/emqx"
-
-```
-
-4. Put the downloaded zip package in the `releases` directory under the EMQX installation directory:
-
-```bash
-
-$ cp emqx-4.2.1.zip ${EMQX_ROOT_DIR}/releases/
-
-```
-
-5. Upgrade to the specified version:
-
-```bash
-
-$ emqx upgrade 4.2.1
-
-Release 4.2.1 not found, attempting to unpack releases/emqx-4.2.1.tar.gz
-Unpacked successfully: "4.2.1"
-Installed Release: 4.2.1
-Made release permanent: "4.2.1"
-```
+## Download the EMQX Zip Package for Release Hot Upgrade
 
-6. Check the version list again, and the status of the previous version will become `old`:
-
-```bash
-
-$ emqx versions
-
-Installed versions:
-* 4.2.1	permanent
-* 4.2.0	old
-```
-
-## Manually permanent after upgrade
-
-The above `emqx upgrade 4.2.1` command actually performs three actions:
-
-- `unpack`
-- `install`
-- `permanent`
-
-After permanent, this version upgrade will be fixed, which means that after the hot upgrade, if EMQX restarts, the new version after the upgrade will be used.
-
-If you don't want to persist while upgrading, you can use the `--no-permanent` parameter:
-
-```bash
-
-$ emqx upgrade --no-permanent 4.2.1
-
-Release 4.2.1 not found, attempting to unpack releases/emqx-4.2.1.tar.gz
-Unpacked successfully: "4.2.1"
-Installed Release: 4.2.1
-
-```
-
-At this time, the version has been successfully upgraded to 4.2.1. However, if you restart EMQX, it will revert to the old version 4.2.0.
-Now, if you check the version list, you will find that the state of 4.2.1 is `current`, not the permanent version:
-
-```bash
-
-$ emqx versions
+{% emqxce %}
+Go to [Download EMQX](https://www.emqx.com/en/downloads?product=broker) for the version and the OS you want, and then select the **"zip"** in the "Install Method".
```

Comment:
should we link it to the .io site?

Reply:
I didn't know there's another website (emqx.io) for the user to download emqx packages. 

After a quick glance of the `https://www.emqx.io/downloads?os=Docker` page, I think the one on `emqx.com` is simpler to the user (use drop-down list than tabs and sub-tabs).